### PR TITLE
Made the abnos names show in crew monitor 

### PR DESCRIPTION
**Pull request by UnokiAs**
## About The Pull Request
Crew monitor now show in what containment zone people are.
image

## Why It's Good For The Game
Clerk gaming will be maximum now
Saving stupid insane agent
Forgotten in cells.

## Changelog
:cl: Unoki
tweak: Crew Monitor now show the the abno names
/:cl:

### CODE REVIEW

File: code/game/objects/effects/spawners/abnormality_room.dm [hunk 1]
```diff
@@ -25,11 +25,13 @@
...
 		AC.datum_reference = abno_datum
 		break
+	for(var/area/containment_zone/ACR)
+		ACR.name = "CZ: [abno_datum.name] ([THREAT_TO_NAME[abno_datum.threat_level]])"
 	for(var/obj/machinery/door/airlock/AR in allObjects)
 		AR.name = "[abno_datum.name] containment zone"
 		AR.desc = "Containment zone of [abno_datum.name]. Threat level: [THREAT_TO_NAME[abno_datum.threat_level]]."
 	for(var/obj/machinery/camera/ACM in allObjects)
-		ACM.c_tag = "Containment zone: [abno_datum.name]"
+		ACM.c_tag = "CZ: [abno_datum.name] ([THREAT_TO_NAME[abno_datum.threat_level]]) "
```
Comment (EgorDinamit) [inline code/game/objects/effects/spawners/abnormality_room.dm:34]: But why? :thinking:

File: code/game/objects/effects/spawners/abnormality_room.dm [hunk 2]
```diff
@@ -25,11 +25,13 @@
 	for(var/obj/machinery/computer/abnormality/AC in allObjects)
 		AC.datum_reference = abno_datum
 		break
+	for(var/area/containment_zone/ACR)
+		ACR.name = "CZ: [abno_datum.name] ([THREAT_TO_NAME[abno_datum.threat_level]])"
```
Comment (EgorDinamit) [inline code/game/objects/effects/spawners/abnormality_room.dm:29]: I assume you didn't test this much, huh? You are iterating through nothing.
You should simply use get_area() proc to get the area in place and then work with it.

File: code/game/objects/effects/spawners/abnormality_room.dm (hunk 1 above)
Comment (UnokiAs) [reply code/game/objects/effects/spawners/abnormality_room.dm:34]: Text was too big and clipped out in both tablet/ Advanced camera selection

File: code/game/objects/effects/spawners/abnormality_room.dm (hunk 2 above)
Comment (UnokiAs) [reply code/game/objects/effects/spawners/abnormality_room.dm:29]: Oups yeah I see it now it stay on the first abno that spawn and keep the name aaaa

**Comment by UnokiAs** [comment]
Will maybe comeback to it later

**Pull request closed without merging**